### ISSUE TITLE
[Alterac Valley] Constant crashes after killing Snivvle

### ISSUE THREAD

**Issue by vilzu** [SunwellWoW/Sunwell-TBC-Bugtracker#3884]
Description: I sneaked in as a subtlety rogue and assassinated Snivvle. Instantly after he was dead, i got disconnect. I cleared my cache and tried to log back in, but the game keep disconnecting me right after login.

How it works: 

How it should work: 

Source (you should point out proofs of your report, please give us some source):

**Comment by 0vertime** [comment]
Yep, took the mines. Now getting thrown at login screen second after logging in.

**Comment by baade123** [comment]
same, guess it was a blood sacrifice

**Comment by MadderThenMad** [comment]
Got it today after taking the mine
